src/pages/umi/guides/serializing-and-deserializing.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Serializing, Deserializing, and sending Transactions
+metaTitle: Serializing, Deserializing, and sending Transactions
+description: Learn how to use the Noop signer to pass partially signed transaction from the Backend to the Frontend (and viceversa)
```

Comment:
Can we change the description, this isn't a noop signer guide but a serialization and deserialization transaction guide. Noop signer is just a byproduct of that.

Reply:
> Learn how to serialization and deserialize transactions with the Metaplex Umi framework to send transactions between different enviroments such as backend and frontend before sending.

something like that